### PR TITLE
General refactorization

### PR DESCRIPTION
**Pull request by jaleszek**

### CODE REVIEW

File: app/helpers/form_answer_helper.rb [hunk 1]
```diff
@@ -59,4 +59,9 @@ def press_summary_updated_by(form_answer)
       "Updated by #{ps.authorable.decorate.full_name} - #{format_date(ps.updated_at)}"
     end
   end
+
+  def assessors_collection_for_bulk
+    assessors = Assessor.available_for(category_picker.current_award_type).map { |a| [a.full_name, a.id] }
```
Comment (matthewford) [inline app/helpers/form_answer_helper.rb]: Line is too long. [106/100]

File: app/models/reports/data_pickers/form_document_picker.rb [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    end
+  end
+
+  def date_started_trading
+    day = doc("started_trading_day")
+    month = doc("started_trading_month")
+    year = doc("started_trading_year")
+
+    if year && month && day
+      Date.new(year.to_i, month.to_i, day.to_i).strftime("%m/%d/%Y") rescue nil
```
Comment (matthewford) [inline app/models/reports/data_pickers/form_document_picker.rb]: Avoid using `rescue` in its modifier form.

**Comment by jaleszek** [comment]
@RuslanHamidullin would you check please ?

**Comment by RuslanHamidullin** [comment]
Good :+1:

**Pull request merged**